sequence-classification/classifiers/hmm_classifier.py

```diff
@@ -22,7 +22,11 @@ def get_transformer(self):
 
 class HMMTransformer(SequenceTransformer):
     def __init__(self):
-        self.vectorizer = CountVectorizer()
+        self.vectorizer = CountVectorizer(analyzer="word",
+                                 tokenizer=None,
+                                 preprocessor=None,
+                                 stop_words=None,
+                                 token_pattern='\\b\\w+\\b')
```

Comment:
Jak dodasz `r` przed stringiem (raw string), to nie musisz cytować backslasha:
`token_pattern='\\b\\w+\\b'` -> `token_pattern=r'\b\w+\b'`